queue_processors/queue_processor/queueproc_utils/script_utils.py

```diff
@@ -0,0 +1,79 @@
+# ############################################################################### #
+# Autoreduction Repository : https://github.com/ISISScientificComputing/autoreduce
+#
+# Copyright &copy; 2021 ISIS Rutherford Appleton Laboratory UKRI
+# SPDX - License - Identifier: GPL-3.0-or-later
+# ############################################################################### #
+"""
+Module that provides script utils
```

Comment:
I'm conscious of the size of this PR growing, but I think this whole module should be combined into the `ReductionScript` class in `reduction_service.py` Though that could be opened as an issue after this and QP refactor

Reply:
I will look into it - they do look like they're doing the same thing, more or less.

Reply:
Actually on a second look I won't do it in this PR as the ReductionScript itself needs a small change to work with `reduce_vars.py`. I think it's fine for now but they'll need to be merged in.